<launch>

    <!-- Robot Agruments -->
    <arg name="ROBOT_IP"       default="192.168.0.2"/>
    <arg name="enable_gripper" default="False"/>

    <!-- RTDE - Position Controller -->
    <node pkg="rtde_controller" type="position_controller" name="ur_position_controller" output="screen">
        <param name="ROBOT_IP"       value="$(arg ROBOT_IP)"/>
        <param name="enable_gripper" value="$(arg enable_gripper)"/>
    </node>

</launch>
